hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -119,14 +119,59 @@ On earlier models the serial number is used as the prefix, however, on Pi4 the M
 * 0 - Use the serial number e.g. "9ffefdef/"
 * 1 - Use the string specified by TFTP_PREFIX_STR
 * 2 - Use the MAC address e.g. "DC-A6-32-01-36-C2/"
-Default: 0
+Default: 0  
 Version: stable/pieeprom-2020-01-17.bin  
 
 ### TFTP_PREFIX_STR
 Specify the custom directory prefix string used when TFTP_PREFIX is set to 1. For example:- TFTP_PREFIX_STR=tftp_test/
-Default: ""
+Default: ""  
 Version: stable/pieeprom-2020-01-17.bin  
 
+### PXE_OPTION43
+Overrides the PXE Option43 match string with a different string. It's normally better to apply customizations to the DHCP server than change the client behaviour but this option is provided in case that's not possible.
+Default: "Raspberry Pi Boot"  
+Version: stable/pieeprom-2020-03-19.bin  
+
+### DHCP_OPTION97
+In earlier releases the client GUID (Option97) was just the serial number repeated 4 times. By default, the new GUID format is
+the concatenation of the fourcc for RPi4 (0x80017590), the board revision (e.g. 0x00c03111) (4-bytes), the least significant 4 bytes of the mac address and the 4-byte serial number.
+This is inteded to be unique but also provide structured information to the DHCP server, allowing Raspberry Pi4 computers to be identified without relying upon the Ethernet MAC OUID.
+
+Specify DHCP_OPTION97=0 to revert the the old behaviour.
+
+Default: 0x80017590
+Version: stable/pieeprom-2020-03-19.bin  
+
+### Static IP address configuration
+If TFTP_IP and the following options are set then DHCP is skipped and the static IP configuration is applied.
+
+#### CLIENT_IP
+The IP address of the client e.g. 192.168.0.1
```

Comment:
I know these are only examples, but to avoid confusion I wonder if it's worth giving different example IP-addresses for CLIENT_IP and GATEWAY ? :man_shrugging:

Reply:
Also, the documentation of TFTP_IP includes the phrase "dotted decimal ip address", so I wonder if it's worth including it here too for clarity?

Reply:
Yes, copy and paste the CLIENT_IP should be something different. I'd thing duplicating dotted decimal everywhere is a bit verbose, and if you know enough to setup a static ip address configuration then it should be obvious